engine/src/org/pentaho/di/trans/TransMeta.java

```diff
@@ -2878,7 +2880,7 @@ public void loadXML( Node transnode, String fname, IMetaStore metaStore, Reposit
             }
           }
         }
-        setPrivateTransformationDatabases( privateTransformationDatabases );
+        setPrivateDatabases( privateTransformationDatabases );
```

Comment:
Yep, here is that PR: https://github.com/pentaho/pentaho-kettle/pull/1634 (a fortnight ago)